### PR TITLE
aotf: toolbar mutate button

### PR DESCRIPTION
**Pull request by oliver-sanders**
addresses: #339 
built on: #544 
related sibling pr: https://github.com/cylc/cylc-uiserver/pull/175

Add a button in the workflow toolbar that lists workflow mutations.

kap

**Requirements check-list**
- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [x] Contains logically grouped changes (else tidy your branch by rebase).
- [x] Does not contain off-topic changes (use other PRs for other changes).
- [x] Appropriate tests are included (unit and/or functional).
- [x] Appropriate change log entry included.
- [x] No documentation update required.

### CODE REVIEW

File: src/utils/aotf.js [hunk 1]
```diff
@@ -44,11 +49,17 @@ import TaskState from '@/model/TaskState.model'
...
   kill: mdiCloseCircle,
+  message: mdiEmail,
+  play: mdiPlay,
   poll: mdiRefreshCircle,
+  release: mdiPlay,
+  reload: mdiReload,
   remove: mdiDelete,
+  setOutputs: mdiGraph,
+  stop: mdiStop,
   trigger: mdiCursorPointer
```
Comment (oliver-sanders) [inline src/utils/aotf.js:63]: Add icons for the workflow-level mutations that didn't show up before.

**Comment by kinow** [comment]
@oliver-sanders is it also closing #135 ?

**Comment by oliver-sanders** [comment]
Closes the first point of #135, the second point is about tracking mutations until they have completed which is something we can't yet do at the backend.

**Review by kinow — APPROVED** [review]
Working like a charm! Code looks excellent.

@oliver-sanders `CHANGES.md` has conflicts now. Also, for a follow-up:

image

I have not submitted this form yet. But apparently the state is kept somewhere I think. So after you submit a mutation once, the form will forever say "submitted", which is a little confusing.

**Comment by hjoliver** [comment]
(Change log deconflicted via GH UI).

**Review by hjoliver — APPROVED** [review]
What @kinow said :tada:   Merging, can fix the "submitted" indicator in a follow-up.

**Pull request merged**